testdata/protobuf/generate.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# generate compiled binary protodescriptors, protoc required then commit files for test usage
+# 
+set -e
+
+THIS_DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
```

Comment:
Quotes around `"$(...)"`

Also when referencing a variable, I always recommend i.e. `${THIS_DIR}` instead of `$THIS_DIR` but it shouldn't matter below.